templates/quickstart-jira-dc-with-vpc.template.yaml

```diff
@@ -567,7 +567,7 @@ Parameters:
     Type: String
   KeyPairName:
     ConstraintDescription: Must be the name of an existing EC2 Key Pair.
-    Description: Public/private EC2 Key Pairs to allow you to securely access the Bastion host
+    Description: Public/private EC2 Key Pairs to allow you to securely access the Bastion host (Type Key Pairs name without file extension).
```

Comment:
Your change is good however there is another option to make sentence shorter:

Public/private EC2 Key Pairs(without the file extension) to allow you to securely access the Bastion host.

Reply:
Thanks @nghazali  description updated

Reply:
Can we also include the change as part of `ConstraintDefinition` for the property something like...

```
ConstraintDescription: Must be the name of an existing EC2 Key Pair. Note the supplied value must not include the file extension.
```

Reply:
@errcode1202 good idea. updated